stack/rush-stack/src/cli/BuildAction.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { CommandLineAction } from  '@microsoft/ts-command-line';
+import { RushStackCommandLine } from './RushStackCommandLine';
+import { BasicTasks } from '../logic/BasicTasks';
+import { BuildContext } from '../logic/BuildContext';
+
+export class BuildAction extends CommandLineAction {
+  constructor(parser: RushStackCommandLine) {
+    super({
+      actionVerb: 'build',
+      summary: 'Build the current project',
+      documentation: ''
+    });
+  }
+
+  protected onDefineParameters(): void { // override
+  }
+
+  protected onExecute(): Promise<void> { // override
+    const buildContext: BuildContext = new BuildContext();
+
+    BasicTasks.doClean(buildContext);
```

Comment:
>BasicTasks.doClean(buildContext); [](start = 4, length = 33)

This should be an option.

Reply:
Out of scope for this PR

---
In reply to: [179336030](https://github.com/Microsoft/web-build-tools/pull/602#discussion_r179336030) [](ancestors = 179336030)